<ui version="4.0" >
 <class>ChangePassword</class>
 <widget class="QDialog" name="ChangePassword" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>修改密码</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" >font: 14pt "宋体";
background-color: rgb(231, 240, 245);</string>
  </property>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>104</y>
     <width>96</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet" >
    <string notr="true" >font: 75 12pt "新宋体";
color: rgb(4, 30, 103);
background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 0), stop:0.5 rgba(224, 234, 245, 0), stop:1 rgba(255, 255, 255, 0));
    border-width:0px;</string>
   </property>
   <property name="text" >
    <string>确认新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPasswordNewConfirm" >
   <property name="geometry" >
    <rect>
     <x>114</x>
     <y>109</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet" >
    <string notr="true" >background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>381</x>
     <y>104</y>
     <width>16</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string notr="true" >color: rgb(255, 0, 0);</string>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>57</y>
     <width>80</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet" >
    <string notr="true" >font: 75 12pt "新宋体";
color: rgb(4, 30, 103);
background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 0), stop:0.5 rgba(224, 234, 245, 0), stop:1 rgba(255, 255, 255, 0));
    border-width:0px;</string>
   </property>
   <property name="text" >
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPasswordNew" >
   <property name="geometry" >
    <rect>
     <x>114</x>
     <y>62</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet" >
    <string notr="true" >background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>381</x>
     <y>57</y>
     <width>16</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string notr="true" >color: rgb(255, 0, 0);</string>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet" >
    <string notr="true" >font: 75 12pt "新宋体";
color: rgb(4, 30, 103);
background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 0), stop:0.5 rgba(224, 234, 245, 0), stop:1 rgba(255, 255, 255, 0));
    border-width:0px;</string>
   </property>
   <property name="text" >
    <string>原密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPasswordOld" >
   <property name="geometry" >
    <rect>
     <x>114</x>
     <y>15</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet" >
    <string notr="true" >background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>381</x>
     <y>10</y>
     <width>16</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string notr="true" >color: rgb(255, 0, 0);</string>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEnter" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>151</y>
     <width>77</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string notr="true" >background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt "汉仪中黑简";
color: rgb(7, 100, 143);</string>
   </property>
   <property name="text" >
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>151</y>
     <width>77</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string notr="true" >background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(214, 240, 253, 255), stop:0.536723 rgba(216, 239, 253, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 2px;
    border-color: rgb(0, 113, 173);
    border-style: solid;
    border-radius:2;
    padding: 2px;

font: 13pt "汉仪中黑简";
color: rgb(7, 100, 143);</string>
   </property>
   <property name="text" >
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
